backend/tests/unit/modules/load/test_load.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from modules.load.domain import commands as domain_commands
+from modules.load.domain.ports import units_of_work
+from modules.load.services import commands as service_commands
+from modules.transform.domain import value_objects as transform_value_objects
+
+from ....common.annotations import YieldFixture
+from .fakers import TestDataDomainRepository, TestSaveDataUnitOfWork
+
+
+def test_data_saved_successfully(
+    test_data_unit_of_work: units_of_work.AbstractDataUnitOfWork,
+):
+    # Given
+    output_data = [
+        transform_value_objects.OutputData(
+            full_name="Jessica Barnes", age=58, is_satisfied=False
+        ),
+    ]
+    domain_command = domain_commands.SaveData(output_data=output_data)
+
+    # When
+    service_commands.save(unit_of_work=test_data_unit_of_work, command=domain_command)
+
+    # Then
+    assert len(test_data_unit_of_work.data.data) == 1  # type: ignore[attr-defined]
+    assert all(
+        [
+            output_data[0].full_name == test_data_unit_of_work.data.data[0].full_name,  # type: ignore[attr-defined]
```

Comment:
Is it required to use [0]? Can't it be checked for whole series?

Reply:
I need to compare output_data elements with unit_of_work.data elements. 
I propose using for loop with index here

for index, item in output_data:
   output_data[index].age == test_data_unit_of_work.data.data[index].age and so on...
   
   
please let me know if it's ok

Reply:
Generaly, id there is a set/list all element should be checked. Not only first one.

I would expect that test check all of them.